### PR TITLE
VZ-5754 - Update release notes for 1.2.1

### PR DESCRIPTION
**Pull request by mgianatagh**
Update release notes for 1.2.1

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+
+### v1.2.1
+Fixes:
+
+- Fixed upgrade issue with private registry.
+- Fixed bug of the public image of WebLogic Monitoring Exporter being used in a private registry configuration.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:12]: bug of the -> bug involving the
Another way to word all the "Fixed bug" entries (which I recommend) is Resolved an issue with... (or Resolved issue with)

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -4,6 +4,26 @@ linkTitle: Release Notes
...
+- Fixed intermittent upgrade failure while upgrading from Verrazzano v1.0.2 to v1.2.0.
+- Fixed console UI error when viewing WebLogic applications.
+- Fixed console UI error when displaying an application that is targeted to a managed cluster that has not completed the registration process.
+- Fixed console UI issue of not displaying the traits for an OAM application.
+- Fixed bug of verrazzano-application-operator pod continually crashing and restarting.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: verrazzano-application-operator -> `verrazzano-application-operator` (code font?)

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -4,6 +4,26 @@ linkTitle: Release Notes
...
+- Fixed bug of the WebLogic workload logHome value being ignored and always using /scratch/log.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: logHome value -> `logHome` value (code font)
/scratch/log -> `/scratch/log` (code font)

**Review by stevshap — APPROVED** [review]

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -4,6 +4,26 @@ linkTitle: Release Notes
...
+- Fixed bug of Prometheus not scraping metrics in Verrazzano managed namespaces that do not have Istio injection enabled.
+- The Verrazzano operators no longer have watches on resources in the kube-system namespace.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:20]: kube-system -> `kube-system` (code font)

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Review by rosemarymarano — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Review by kminder — APPROVED** [review]
LGTM

**Pull request merged**